episodes/wrap-up3.md

```diff
@@ -20,10 +20,66 @@ After completing this episode, participants should be able to...
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+We have reached the end of this training.
+
+## Looking Back
+
+Throughout this training we promoted a collaborative approach to lesson development and maintenance (as many other initiatives in The Carpentries community are conducted). Collaboration helps bring different skills and perspectives to a project and share the load of the work required to bring the project successfully past the finishing line. For lesson projects, this shared effort also extends to maintaining and updating the lesson after the initial development phase.
+
+::::::::::::: challenge
```

Comment:
Maybe this should be a discussion block instead of a challenge?  I'm not quite sure myself.